### ISSUE TITLE
cannot find package "github.com/gobuffalo/mw-contenttype"

### ISSUE THREAD

**Issue by Sispheor** [gobuffalo/buffalo#1612]
## Description

After a fresh install on Linux, when running `buffalo dev` after creating a fresh API project. I have missing lib error.
I haven't seen in the doc other step of installation.

### Steps to Reproduce the Problem

1. Install buffalo on Linux with sqlite support `go get -u -v -tags sqlite github.com/gobuffalo/buffalo/buffalo`
1. create a fresh project `buffalo new --api coke`
1. run `buffalo dev`

### Expected Behavior

The app run in dev mode

### Actual Behavior

```
 buffalo dev
buffalo: 2019/03/03 01:22:20 === Rebuild on: :start: ===
buffalo: 2019/03/03 01:22:20 === Running: go build -v -i -tags development sqlite -o tmp/piclodio4-build  (PID: 79956) ===
actions/app.go:11:2: cannot find package "github.com/gobuffalo/mw-contenttype" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-contenttype (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-contenttype (from $GOPATH)
actions/app.go:6:2: cannot find package "github.com/gobuffalo/mw-forcessl" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-forcessl (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-forcessl (from $GOPATH)
actions/app.go:7:2: cannot find package "github.com/gobuffalo/mw-paramlogger" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-paramlogger (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-paramlogger (from $GOPATH)
actions/app.go:13:2: cannot find package "github.com/rs/cors" in any of:
	/usr/local/go/src/github.com/rs/cors (from $GOROOT)
	/home/nico/go/src/github.com/rs/cors (from $GOPATH)
actions/app.go:8:2: cannot find package "github.com/unrolled/secure" in any of:
	/usr/local/go/src/github.com/unrolled/secure (from $GOROOT)
	/home/nico/go/src/github.com/unrolled/secure (from $GOPATH)
buffalo: 2019/03/03 01:22:20 === Error! ===
buffalo: 2019/03/03 01:22:20 === exit status 1
actions/app.go:11:2: cannot find package "github.com/gobuffalo/mw-contenttype" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-contenttype (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-contenttype (from $GOPATH)
actions/app.go:6:2: cannot find package "github.com/gobuffalo/mw-forcessl" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-forcessl (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-forcessl (from $GOPATH)
actions/app.go:7:2: cannot find package "github.com/gobuffalo/mw-paramlogger" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-paramlogger (from $GOROOT)
	/home/nico/go/src/github.com/gobuffalo/mw-paramlogger (from $GOPATH)
actions/app.go:13:2: cannot find package "github.com/rs/cors" in any of:
	/usr/local/go/src/github.com/rs/cors (from $GOROOT)
	/home/nico/go/src/github.com/rs/cors (from $GOPATH)
actions/app.go:8:2: cannot find package "github.com/unrolled/secure" in any of:
	/usr/local/go/src/github.com/unrolled/secure (from $GOROOT)
	/home/nico/go/src/github.com/unrolled/secure (from $GOPATH)
 ===
```

### Info

<details>

```
### Buffalo Version
v0.14.0

### App Information
Pwd=/home/nico/go/src/github.com/sispheor/piclodio4
Root=/home/nico/go/src/github.com/sispheor/piclodio4
GoPath=/home/nico/go
PackagePkg=github.com/sispheor/piclodio4
ActionsPkg=github.com/sispheor/piclodio4/actions
ModelsPkg=github.com/sispheor/piclodio4/models
GriftsPkg=github.com/sispheor/piclodio4/grifts
WithModules=false
Name=piclodio4
Bin=bin/piclodio4
VCS=git
WithPop=true
WithSQLite=true
WithDep=false
WithWebpack=false
WithNodeJs=false
WithYarn=false
WithDocker=true
WithGrifts=true
AsWeb=false
AsAPI=true
PackageJSON={map[]}

### Go Version
go version go1.12 linux/amd64

### Go Env
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/nico/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/nico/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build617914827=/tmp/go-build -gno-record-gcc-switches"

### Node Version
Node Not Found

### NPM Version
NPM Not Found

### Yarn Version
Yarn Not Found

### PostgreSQL Version
PostgreSQL Not Found

### MySQL Version
MySQL Not Found

### SQLite Version
SQLite Not Found

### Dep Version
could not find a Gopkg.toml file

### Dep Status
could not find a Gopkg.toml file
```

</details>

**Comment by Sispheor** [comment]
Also, maybe realted,
```
buffalo pop
v4.9.9

[POP] 2019/03/03 01:31:15 warn - unable to load connection development: could not create new connection: sqlite3 support was not compiled into the binary
[POP] 2019/03/03 01:31:15 warn - unable to load connection test: could not create new connection: sqlite3 support was not compiled into the binary
[POP] 2019/03/03 01:31:15 warn - unable to load connection production: could not create new connection: sqlite3 support was not compiled into the binary
```

**Comment by gudata** [comment]
The same here.
I started with the release version of buffalo got 
```
wget https://github.com/gobuffalo/buffalo/releases/download/v0.14.0/buffalo_0.14.0_linux_amd64.tar.gz
```
then create a new app

```
buffalo new   --skip-pop  ap-connect
DEBU[2019-03-04T16:15:24+02:00] Step: 8f85b2f7
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/.codeclimate.yml
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/.env
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/README.md
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/actions/actions_test.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/actions/app.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/actions/home.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/actions/home_test.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/actions/render.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/fixtures/sample.toml
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/grifts/init.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/inflections.json
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/main.go
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/config/buffalo-app.toml
DEBU[2019-03-04T16:15:24+02:00] Step: 37a46a0d
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/.dockerignore
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/Dockerfile
DEBU[2019-03-04T16:15:24+02:00] Step: d9de446c
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/.buffalo.dev.yml
DEBU[2019-03-04T16:15:24+02:00] Step: 9c62cf47
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] Exec: go get github.com/gobuffalo/buffalo-plugins
DEBU[2019-03-04T16:15:24+02:00] Step: 5137de2d
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] File: /home/user/go/src/private-repo/some_project/config/buffalo-plugins.toml
DEBU[2019-03-04T16:15:24+02:00] Step: 08b17b68
DEBU[2019-03-04T16:15:24+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:24+02:00] Exec: go get -t ./...
DEBU[2019-03-04T16:15:26+02:00] Step: c26c70d8
DEBU[2019-03-04T16:15:26+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/actions/app.go
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/actions/home.go
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/actions/home_test.go
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/actions/render.go
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/locales/all.en-us.yaml
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/public/robots.txt
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/templates/_flash.html
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/templates/application.html
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/templates/index.html
DEBU[2019-03-04T16:15:26+02:00] Step: 1e28f841
DEBU[2019-03-04T16:15:26+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:15:26+02:00] LookPath: npm
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/.babelrc
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/assets/css/_buffalo.scss
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/assets/css/application.scss
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/assets/images/favicon.ico
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/assets/images/logo.svg
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/assets/js/application.js
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/package.json
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/public/assets/.keep
DEBU[2019-03-04T16:15:26+02:00] File: /home/user/go/src/private-repo/some_project/webpack.config.js
DEBU[2019-03-04T16:15:26+02:00] LookPath: yarnpkg
DEBU[2019-03-04T16:15:26+02:00] Exec: yarnpkg install --no-progress --save
DEBU[2019-03-04T16:15:26+02:00] yarn install v1.13.0

DEBU[2019-03-04T16:15:26+02:00] info No lockfile found.

DEBU[2019-03-04T16:15:26+02:00] [1/4] Resolving packages...

DEBU[2019-03-04T16:15:55+02:00] [2/4] Fetching packages...

DEBU[2019-03-04T16:15:55+02:00] info fsevents@1.2.7: The platform "linux" is incompatible with this module.

DEBU[2019-03-04T16:15:55+02:00] info "fsevents@1.2.7" is an optional dependency and failed compatibility check. Excluding it from installation.

DEBU[2019-03-04T16:15:55+02:00] [3/4] Linking dependencies...

DEBU[2019-03-04T16:15:57+02:00] [4/4] Building fresh packages...

DEBU[2019-03-04T16:17:35+02:00] success Saved lockfile.

DEBU[2019-03-04T16:17:35+02:00] Done in 129.43s.

DEBU[2019-03-04T16:17:35+02:00] File: /home/user/go/src/private-repo/some_project/templates/application.html
DEBU[2019-03-04T16:17:35+02:00] Step: c3ddc0f7
DEBU[2019-03-04T16:17:35+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:17:35+02:00] Step: c9024ec8
DEBU[2019-03-04T16:17:35+02:00] Chdir: /home/user/go/src/private-repo/some_project
DEBU[2019-03-04T16:17:35+02:00] File: /home/user/go/src/private-repo/some_project/.gitignore
DEBU[2019-03-04T16:17:35+02:00] Exec: git init
Initialized empty Git repository in /home/user/go/src/private-repo/some_project/.git/
DEBU[2019-03-04T16:17:35+02:00] Exec: git add .
DEBU[2019-03-04T16:17:35+02:00] Exec: git commit -q -m Initial Commit
INFO[2019-03-04T16:17:35+02:00] Congratulations! Your application, ap-connect, has been successfully built!
INFO[2019-03-04T16:17:35+02:00] You can find your new application at: /home/user/go/src/private-repo/some_project
INFO[2019-03-04T16:17:35+02:00] Please read the README.md file in your new application for next steps on running your application.
```


After run the dev command, i saw the error
```
buffalo dev
buffalo: 2019/03/04 16:21:02 === Rebuild on: :start: ===
buffalo: 2019/03/04 16:21:02 === Running: go build -v -i -tags development -o tmp/ap-connect-build  (PID: 26023) ===
actions/app.go:11:2: cannot find package "github.com/gobuffalo/mw-i18n" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-i18n (from $GOROOT)
	/home/guda/go/src/github.com/gobuffalo/mw-i18n (from $GOPATH)
buffalo: 2019/03/04 16:21:02 === Error! ===
buffalo: 2019/03/04 16:21:02 === exit status 1
actions/app.go:11:2: cannot find package "github.com/gobuffalo/mw-i18n" in any of:
	/usr/local/go/src/github.com/gobuffalo/mw-i18n (from $GOROOT)
	/home/guda/go/src/github.com/gobuffalo/mw-i18n (from $GOPATH)
 ===

webpack is watching the files…

Live Reload listening on port 35729

Hash: 43935350eaa1b6775490
Version: webpack 4.5.0
Time: 2137ms
Built at: 3/4/2019 4:21:05 PM
                               Asset       Size       Chunks             Chunk Names
674f50d287a8c48dc19ba404d20fe713.eot    162 KiB               [emitted]  
application.2837a98680f18fd6cdf6.css   1.17 MiB  application  [emitted]  application
 application.43935350eaa1b6775490.js    851 KiB  application  [emitted]  application
                  images/favicon.ico   1.12 KiB               [emitted]  
                     images/logo.svg   64.4 KiB               [emitted]  
                       manifest.json  270 bytes               [emitted]  
Entrypoint application = application.2837a98680f18fd6cdf6.css application.43935350eaa1b6775490.js
   [0] multi ./node_modules/jquery-ujs/src/rails.js ./assets/css/application.scss ./assets/js/application.js 52 bytes {application} [built]
[./assets/css/application.scss] 39 bytes {application} [built]
[./assets/js/application.js] 126 bytes {application} [built]
[./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {application} [built]
    + 11 hidden modules
Child mini-css-extract-plugin node_modules/css-loader/index.js??ref--4-1!node_modules/sass-loader/lib/loader.js??ref--4-2!assets/css/application.scss:
    Entrypoint mini-css-extract-plugin = *
    [./node_modules/css-loader/index.js??ref--4-1!./node_modules/sass-loader/lib/loader.js??ref--4-2!./assets/css/application.scss] ./node_modules/css-loader??ref--4-1!./node_modules/sass-loader/lib/loader.js??ref--4-2!./assets/css/application.scss 397 KiB {mini-css-extract-plugin} [built]
        + 9 hidden modules

```

after installing manually
```
go get github.com/gobuffalo/mw-i18n
``` 
I can finally load the web ui on port 3000

**Comment by tadeogutierrez** [comment]
same here. That new dependency is not properly added for "buffalo new"

**Comment by tadeogutierrez** [comment]
and running go github.com/gobuffalo/mw-i18n seems to download "a lot" of packages

**Comment by stanislas-m** [comment]
This should be fixed now in v0.14.5.
